### PR TITLE
Add GPU counter page for IMG

### PR DESCRIPTION
**Pull request by rschiu**

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ toc:
         url: docs/gpu-counters/arm
       - title: Qualcomm® Adreno™ Counters
         url: docs/gpu-counters/qualcomm
+      - title: IMG® Counters
```
Comment (jakon2048) [inline _data/nav.yml:22]: Imagination® PowerVR™

File: gpu-counters.md [hunk 2]
```diff
@@ -9,3 +9,4 @@ details on the available counters of your device's GPU.
 
 * [ARM Counters](arm)
 * [Qualcomm Counters](qualcomm)
+* [IMG Counters](img)
```
Comment (jakon2048) [inline gpu-counters.md:12]: Imagination Counters

**Review by jakon2048 — CHANGES_REQUESTED** [review]
Requested some changes, otherwise lgtm

**Review by pau-baiget — APPROVED** [review]
LGTM!!

File: gpu-counters.md (hunk 2 above)
Comment (rschiu) [reply gpu-counters.md]: Done

File: _data/nav.yml (hunk 1 above)
Comment (rschiu) [reply _data/nav.yml]: Done

**Review by jakon2048 — APPROVED** [review]

**Pull request merged**